plugins/google-sheets/src/pages/SelectSheet.tsx

```diff
@@ -23,17 +24,22 @@ export function SelectSheetPage({ onError, onSheetSelected }: Props) {
         }
     }, [isSpreadSheetInfoError, onError])
 
-    const handleSheetSelect = (e: SelectChangeEvent) => {
-        setSelectedSheetId(e.target.value)
+    const handleSheetSelect = (event: SelectChangeEvent) => {
+        const id = parseInt(event.currentTarget.value)
```

Comment:
Leave as a string maybe? Especially considering that later it gets stringified a few times anyway? Who knows if these numbers actually fit into 53 bits or not.

Reply:
Hmm yeah feels a bit messy, let me try change that

Reply:
For future readers, we reverted it because Google does return these as numbers, so it's fine.